release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -238,6 +238,68 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+==== Beta APIs removed from Kubernetes v1.22
+
+The upgrade to Kubernetes v1.22 removed the following deprecated
+`v1beta1` APIs. Migrate manifests and API clients to use the `v1` API
+version. For more information on migrating deprecated APIs, see the
+https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22[Kubernetes
+documentation].
+
+.`v1beta1` APIs removed from Kubernetes v1.22
+[cols=",,",options="header",]
+|===
+|Feature |API |Notable Changes
+|Webhook resources
+|`admissionregistration.k8s.io/v1beta1`
+|https://kubernetes.io/docs/reference/using-api/deprecation-guide/#webhook-resources-v122[Yes]
+
+|CustomResourceDefinition
+|`apiextensions.k8s.io/v1beta1`
+|https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122[Yes]
+
+|APIService
+|`apiregistration.k8s.io/v1beta1`
+|No
+
+|SubjectAccessReview resources
+|`authorization.k8s.io/v1beta1`
+|https://kubernetes.io/docs/reference/using-api/deprecation-guide/#subjectaccessreview-resources-v122[Yes]
+
+|CertificateSigningRequest
+|`certificates.k8s.io/v1beta1`
+|https://kubernetes.io/docs/reference/using-api/deprecation-guide/#certificatesigningrequest-v122[Yes]
+
+|Lease
+|`coordination.k8s.io/v1beta1`
+|No
+
+|Ingress
+|`extensions/v1beta1`
+|https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122[Yes]
+
+|
+|`networking.k8s.io/v1beta1`
```

Comment:
I don't think this would be good for accessibility. I would consider combining it with the `extensions/v1beta1` cell above, or filling out the first and last columns in this row.